esrc/rescueoverview.php

```diff
@@ -473,4 +483,86 @@ function displayNotes($row, $isCoord = 0, $isSARAgent = 0)
 		}
 	}
 }
+
+/**
+ * Check killboard and update rescue status if activity detected
+ * @param unknown $data
+ */
+function checkKbActivity($data) {
+
+	// create a new database connection
+	$database = new Database();
```

Comment:
Is a separate db connection necessary for the check? Can the db created before can be used?